app/controllers/api/answers_controller.rb

```diff
@@ -30,6 +30,7 @@ def create
     @answer.user = current_user
     if @answer.save
       Newspaper.publish(:answer_create, @answer)
+      Newspaper.publish(:answer_destroy, @answer)
```

Comment:
`if @answer.save`の処理で`:answer_destroy`となると、回答の削除に関することがなぜここに…？と思われる方もいるかな〜と思いました👀`config/initializers/newspaer.rb`に記載されている他のモデルの処理が参考になるかと思うんですが
```suggestion
      Newspaper.publish(:answer_save, @answer)
```
となるように`config/initializers/newspaer.rb`と併せて名づけを調整した方がいいかもな…と思ったりしました🙏